inductiva/fluids/scenarios/wind_tunnel/README.md

```diff
@@ -0,0 +1,151 @@
+# Wind Tunnel Scenario
+
+This scenario models the aerodynamics of an object inside a virtual wind tunnel for a given airflow velocity. Air is injected on a side wall of the wind tunnel, the flow changes according to the structure of the object and leaves through an outlet on the other side. The system is modelled with the steady-state equations for incompressible flow and the $k-\epsilon$ turbulence models.
```

Comment:
Split the middle sentence into two, e.g. "Air is injected on a side wall of the wind tunnel and leaves through an outlet on the opposite side. The air flow within the tunnel is modified according to the structure of the object."